web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Integrator.AuthController do
+  use Integrator.Web, :controller
+
+  def index(conn, _params) do
+    redirect conn, external: Slack.authorize_url!(scope: "identity.basic")
+  end
+
+  def callback(conn, params) do
```

Comment:
I guess the params are different in this case, so we could just use pattern matching and define two functions